docs/user-guide/nextcloud-integration/README.md

```diff
@@ -0,0 +1,85 @@
+
+## How can I use this integration?
+
+As a user of both Nextcloud and OpenProject, you can benefit from this integration in both platforms:
+
+- **In OpenProject:**
+  The Files tab lets you see regular attachments to a work package but now also lists files that had been linked from Nextcloud to OpenProject. You can use secondary actions available with each file to directly open it, download it, show the containing folder in Nextcloud or remove the link.
+  ![Empty_status_files](1_0_00-No_files_linked.png)
+
+  ![OP_linked_files](1_1_00-All_files_available.png)
+
+  **IMPORTANT NOTE:** In this version of the integration, linking Nextcloud files via OpenProject is not possible. This will be possible in the near future.
+
+  - **In Nextcloud:**
+    You will be able to access the OpenProject tab in Nextcloud by accessing the details split screen of any file. In this tab, you will be able to search for the work package to which you would like to link the current file. Once a work package is linked to a file, you can always unlink it  by clicking on the **unlink** icon.
+
+    ![NC_search_WP](0_0_00-File_Relation_Search.png)
+
+    ![NC_linked_WP](0_1_01-File_WP_Actions.png)
+
+    
+    In addition to actions related to the file itself, you can also display the OpenProject widge in the Nextcloud dashboard in order to keep an eye on the the latest changes and updates to your work packages:
+    ![add_NC_widget](Add_OpenProject_widget-0ea8c054.png)
+
+    ![added_NC_widget](Nextcloud_dashboard-c04681eb.png)
+
+    Additionally, you can activate three extra features for your Nextcloud and OpenProject integration via your personal settings under 'Connected accounts': **Enable navigation link** to display a link to your OpenProject instance in the header navigation, **Enable unified search for tickets** to include OpenProject information in the the built-in universal search and **Enable notifications for activity in my work packages** to receive Nextcloud notifications in your OpenProject work packages.
+
+    ![NC_extra_settings](Nextcloud_connected_account-b9ffa0e3.png)
+
+    ![NC_extra_navlink](Navigation_link_OpenProject-0fc98e3b.png)
+
+    ![NC_extra_search](Unified_search-73e2dc96.png)
+
+
+## **Next steps for the integration**
+
+Further integration efforts are under way. In the near future, users will be able to linking and upload files to Nextcloud directly from OpenProject.
+
+## Possible errors and troubleshooting
+
+- **OpenProject - No permissions to see the file:** If the user doesn't have permissions to see the details of a file or to open some of the files linked to this work packages (due to permissions, they will be able to see the name, time of last modification and the name of the modifier but will _not_ be able to perform any further actions. To open or access such files, they will need to contact the Nextcloud administrator or the creator of the file so that they can provide the necessary actions.
```

Comment:
There was a mix of "the user" (third person) and "you" (second person). I've readjusted the text so everything is in third person, but it's a stylistic choice. Can also rewrite for "you" if we prefer the more direct/personal style.

Reply:
Hi @marcalcobe and @psatyal, we directly address of the user in 2nd person in our docs. We should try to keep this consistent.

Reply:
Hi @birthe,
I will update the documentation to use the 2nd person.